noggin/templates/user-settings-agreements.html

```diff
@@ -0,0 +1,51 @@
+{% extends "user-settings.html" %}
+
+{% import '_form_macros.html' as macros %}
+
+{% block settings_content %}
+<div class="list-group">
+    {% for agreement in agreementslist %}
+    <div class="list-group-item">
+        <div class="row align-items-center">
+          <div class="col h6">
+            <div>{{ agreement.name }}</div>
+          </div>
+          <div class="col-auto">
+            {% if user.username in agreement.users %}
```

Comment:
Did you check if we get a list of signed agreements in the user API response? If not, it could be worthwhile to try to have it there.